state/protocol/mutator.go

```diff
@@ -1,52 +0,0 @@
-// (c) 2019 Dapper Labs - ALL RIGHTS RESERVED
-
-package protocol
-
-import (
-	"github.com/onflow/flow-go/model/flow"
-)
-
-// Mutator represents an interface to modify the persistent protocol state in a
-// way that conserves its integrity. It enforces a number of invariants on the
-// input data that ensures the internal bookkeeping mechanisms remain functional
-// and valid.
-type Mutator interface {
```

Comment:
comment for other reviewers:
* replaced by interface https://github.com/onflow/flow-go/blob/32b363388cc2aca861d3a46d23a2e7a1ed801377/state/protocol/state.go#L38